ndmg/utils/qa_utils.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+
+# Copyright 2016 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# qa_utils.py
+# Created by Wilson Tang
```

Comment:
all this boilerplate copyright text is removed in the current `staging-tmp` for most modules (since we have a `LICENSE.txt` in the root, so it doesn't actually do anything legally, and it takes up space and looks ugly)

that being said, it takes approximately 15 seconds to remove, so if you don't feel like doing it it's not exactly a huge drag on our part

that also being said, it takes approximately 15 seconds to remove, so just do that and replace it with text similar to the other modules (see e.g. the top of `staging-tmp`'s `ndmg_bids.py`)

Reply:
this applies to all modules you guys add

Reply:
check commit 09e7135. restructured both qa_tensor and qa_utils for this issue.